shared/constants/fs.tsx

```diff
@@ -1046,3 +1037,33 @@ export const erroredActionToMessage = (
       return errorIsTimeout ? timeoutExplain : 'An unexplainable error has occurred.'
   }
 }
+
+export const erroredActionToMessageWithEdits = (edits: Types.Edits, error: Types.FsError): string => {
+  if (error.erroredAction.type === FsGen.commitEdit) {
+    const edit = edits.get(error.erroredAction.payload.editID) || emptyNewFolder
+    let actionStr = ''
+    switch (edit.type) {
+      case Types.EditType.NewFolder:
+        actionStr = 'make a new folder'
+        break
+      case Types.EditType.Rename:
+        actionStr = 'rename'
+        break
+      default:
+        ;((_: never) => {})(edit.type)
```

Comment:
??????

Reply:
This is to make sure we exhaust all possible values. I'll turn it a function and rely on `return`